src/react/deckgl.js

```diff
@@ -27,7 +27,16 @@ const {Deck, log} = experimental;
 
 const propTypes = Object.assign({}, Deck.propTypes, {
   viewports: PropTypes.array, // Deprecated
-  viewport: PropTypes.object // Deprecated
+  viewport: PropTypes.object, // Deprecated
+
+  // Viewport props (TODO - should only support these on the react component)
+  longitude: PropTypes.number, // The longitude of the center of the map.
+  latitude: PropTypes.number, // The latitude of the center of the map.
+  zoom: PropTypes.number, // The tile zoom level of the map.
+  bearing: PropTypes.number, // Specify the bearing of the viewport
+  pitch: PropTypes.number, // Specify the pitch of the viewport
+  altitude: PropTypes.number, // Altitude of camera. Default 1.5 "screen heights"
+  position: PropTypes.array // Camera position for FirstPersonViewport
```

Comment:
`width` , `height` missing? FlyTo interpolater requires these to calculate correct zoom, can you run `viewport-transitions-flyTo` example and make sure it works fine.